packages/manager/modules/web-paas/src/add/add.controller.js

```diff
@@ -15,6 +10,7 @@ export default class {
     Alerter,
     WebPaas,
     WucOrderCartService,
+    CucRegionService,
```

Comment:
I think we can't use cuc components and services outside public cloud

Reply:
Its already included in the apps of `/web` (web universe) and is also used in modules outside cloud